e2e/passkey/passkeyProxy.test.ts

```diff
@@ -0,0 +1,76 @@
+import { createAndFundKeypair, getNonce } from '../scaffolding/helpers';
+import { KeyringPair } from '@polkadot/keyring/types';
+import { ExtrinsicHelper } from '../scaffolding/extrinsicHelpers';
+import { getFundingSource } from '../scaffolding/funding';
+import { u8aWrapBytes } from '@polkadot/util';
+import { createPassKeyAndSignAccount, createPassKeyCall, createPasskeyPayload } from '../scaffolding/P256';
+const fundingSource = getFundingSource('passkey-proxy');
+
+describe('Passkey Pallet Tests', function () {
+  describe('proxy basic tests', function () {
+    let fundedKeys: KeyringPair;
+    let receiverKeys: KeyringPair;
+
+    beforeEach(async function () {
+      fundedKeys = await createAndFundKeypair(fundingSource, 100_000_000n);
+      receiverKeys = await createAndFundKeypair(fundingSource);
+    });
+
+    it('should fail due to unsupported call', async function () {
+      const accountPKey = fundedKeys.publicKey;
+      const nonce = await getNonce(fundedKeys);
+
+      const remarksCalls = ExtrinsicHelper.api.tx.system.remark('passkey-test');
+      const { passKeyPrivateKey, passKeyPublicKey, passkeySignature } = createPassKeyAndSignAccount(accountPKey);
+      const accountSignature = fundedKeys.sign(u8aWrapBytes(passKeyPublicKey));
+      const passkeyCall = await createPassKeyCall(accountPKey, nonce, accountSignature, remarksCalls);
+      const passkeyPayload = await createPasskeyPayload(passKeyPrivateKey, passKeyPublicKey, passkeyCall, false);
+
+      const passkeyProxy = ExtrinsicHelper.executePassKeyProxy(fundedKeys, passkeyPayload);
+      assert.rejects(passkeyProxy.fundAndSendUnsigned(fundingSource));
+    });
+
+    it('should fail to transfer balance due to bad account ownership proof', async function () {
+      const accountPKey = fundedKeys.publicKey;
+      const nonce = await getNonce(fundedKeys);
+      const transferCalls = ExtrinsicHelper.api.tx.balances.transferAllowDeath(receiverKeys.publicKey, 0n);
+      const { passKeyPrivateKey, passKeyPublicKey, passkeySignature } = createPassKeyAndSignAccount(accountPKey);
+      const accountSignature = fundedKeys.sign('badPasskeyPublicKey');
+      const passkeyCall = await createPassKeyCall(accountPKey, nonce, accountSignature, transferCalls);
+      const passkeyPayload = await createPasskeyPayload(passKeyPrivateKey, passKeyPublicKey, passkeyCall, false);
+
+      const passkeyProxy = ExtrinsicHelper.executePassKeyProxy(fundedKeys, passkeyPayload);
+      assert.rejects(passkeyProxy.fundAndSendUnsigned(fundingSource));
+    });
+
+    it('should fail to transfer balance due to bad passkey signature', async function () {
+      const accountPKey = fundedKeys.publicKey;
+      const nonce = await getNonce(fundedKeys);
+      const transferCalls = ExtrinsicHelper.api.tx.balances.transferAllowDeath(receiverKeys.publicKey, 0n);
+      const { passKeyPrivateKey, passKeyPublicKey, passkeySignature } = createPassKeyAndSignAccount(accountPKey);
```

Comment:
This confused me trying to understand how this process creates a bad passkeySignature, and if `passkeySignature` is bad, why this test fails since that value is apparently never reused in each of these tests.  The passkeySignature here is not what causes the failure it is the bad payload generated by the test function.

Reply:
There are two passKeySignatures, one above `passkeySignature` is passkey signing the account and account signing the passkey, this resembles a registration process on provider side

When we submit the passkey we sign the payload, this test is testing bad passkey signature on transaction time